modules/blockuser.cpp

```diff
@@ -24,6 +24,7 @@ using std::vector;
 class CBlockUser : public CModule {
   public:
     MODCONSTRUCTOR(CBlockUser) {
+        //Creating all commands
```

Comment:
Yes I agree/will take it out. Also would this be worthwhile to work on? I think I should have created an issue first, sorry about this.